streams/data-quality-with-dedupe-and-validation/data-quality-with-dedupe-and-validation.json

```diff
@@ -1,6 +1,6 @@
 {
   "schemaVersion": 1,
-  "name": "Data Quality with Deduplicate and Validation",
+  "name": "Data Quality With Deduplicate And Validation",
```

Comment:
This name will still be to long, can we rephrase it?

Example:
Ensure Data Quality